### DISCUSSION TITLE
Cursor timeline only going so far.

### DISCUSSION THREAD

**Discussion by Bug4B0o** [Avnsx/fansly#150] [Q&A]
I'm trying to download a whole page, but it only gets 17 files from the account out of the hundred or so.

![image](https://github.com/Avnsx/fansly-downloader/assets/54789678/46664d35-31e1-49df-a172-9ac3089ae0f7)
![image](https://github.com/Avnsx/fansly-downloader/assets/54789678/151932ad-f8b2-43eb-8b9a-a00fd2415a06)

**Comment by Avnsx** [comment]
Duplicate of https://github.com/Avnsx/fansly-downloader/issues/148

**Reply by Bug4B0o** [reply]
Ok
